src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java

```diff
@@ -64,30 +68,21 @@ public void testLogInAndLogOut(){
 
 		//Log out
 		navigateMenu.clickLogoutButton();
+		ScreenshotService screen = new ScreenshotService();
+		screen.screenshot();
 		Assert.assertTrue(navigateMenu.isLoginButtonPresent(), "The log in button is not present.");
 	}
 
 	@Test
 	@MethodOwner(owner = "kdenysiuk")
 	public void testDoAPurchase(){
-		//Open DemoBlaze homepage and verify
-		HomePage homePage = new HomePage(getDriver());
-		homePage.open();
-		Assert.assertTrue(homePage.isPageOpened(), "The page is not opened");
-
-		//Open login label
-		NavigateMenu navigateMenu = homePage.getNavigateMenu();
-		Assert.assertTrue(navigateMenu.isUIObjectPresent(), "The navigate menu wasn't found.");
-		Assert.assertTrue(navigateMenu.isLoginButtonPresent(), "The login button wasn't found.");
-		LogInPage logInPage = navigateMenu.clickLogInButton();
-
 		//Log in
-		logInPage.fillUsernameField(R.TESTDATA.get("user_profile"));
-		logInPage.fillPasswordField(R.TESTDATA.get("user_password"));
-		logInPage.clickLoginButton();
-		homePage.open();
+		AuthService login = new AuthService();
+		login.login(R.TESTDATA.get("user_profile"), R.TESTDATA.get("user_password"));
```

Comment:
also can be in one line `new AuthService().login()`

Reply:
The same mistake as was with `screen` name. do not shorten it
use not `login` but `authService`
because login.login(...) is not something clear